### DISCUSSION TITLE
Autosave subtasks

### DISCUSSION THREAD

**Discussion by cpetrovay** [Leantime/leantime#680] [Ideas]
When updating a task in either table or kaban view, the change is automatically applied and saved. However, when changing the status to a subtask, the user must explicitly click the save button for each subtask. Clicking this button causes the page to refresh, meaning any unsaved changes on other subtasks are lost.

Not requiring explicit saving for changes to subtasks would be nice.

**Comment by marcelfolaron** [comment]
Thanks. I agree. I created an issue #687 from this.

**Reply by marcelfolaron** [reply]
Fixed in latest version 2.2.4
